css/secondary-page.styl

```diff
@@ -62,6 +62,7 @@
               background: rgba(255,255,255, 0.2)
               color: rgba(255,255,255,1)
             &.active
+            &[aria-selected=true]
```

Comment:
Dunno, these are the styles that are already in use on the live site. :man_shrugging: